docs/source/admin/log-messages.md

```diff
@@ -1,33 +1,30 @@
 # Interpreting common log messages
 
 When debugging errors and outages, looking at the logs emitted by
-JupyterHub is very helpful. This document tries to document some common
-log messages, and what they mean.
+JupyterHub is very helpful. This document intends to describe some common
+log messages, and what necessary actions to take.
```

Comment:
This section of the docs focus on these three main areas: explain the log message, list a some of of the most common causes that generate this message, and what should be the first steps to take in order to fix this cause.

I believe it makes sense to mention all three of them:

```suggestion
log messages, what they mean and what are the most common causes that generated them, as well as some possible ways to fix them.
```